src/collective/easyform/browser/view.py

```diff
@@ -346,6 +346,22 @@ def __call__(self, fname=None, fset=None):
         return super(EasyFormInlineValidationView, self).__call__(fname, fset)
 
 
+class GetSaveDataAdaptersView(BrowserView):
+
+    def __call__(self, *args, **kwargs):
+        """return all contained save data adapters"""
+        from collective.easyform.api import get_actions
```

Comment:
This one is already imported at top level

Reply:
oops, thx for catching that... I sometimes let old habits from old programming languages make me put the imports where I use them, rather than at the top of the file.